Automattic/jetpack
# Packages: Remove Manager pattern from JITM

As we've decided to move away from the Manager pattern, this PR removes it from the JITM package.

#### Changes proposed in this Pull Request:
* Remove Manager pattern from JITM package

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
Part of the Jetpack DNA project - p1HpG7-70O-p2

#### Testing instructions:
* Checkout the branch.
* Verify the JITM package still works well and you can see JITMs in the admin.

#### Proposed changelog entry for your changes:
* Remove Manager pattern from JITM
